tests/test_terra_notebook_utils.py

```diff
@@ -155,8 +155,35 @@ def test_prepare_merge_workflow_input(self):
         self.assertEqual(expected_input_keys, sorted(input_keys))
         self.assertEqual(expected_output_keys, sorted(output_keys))
 
+    def _test_combine_with_bucket_objs(self):
```

Comment:
Are these tests run?

Reply:
No, they must be run manually.

I've added TODOs to create fixtures for these tests.

Reply:
done